atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1373,6 +1373,12 @@ void WebContents::SetSize(const SetSizeParams& params) {
     guest_delegate_->SetSize(params);
 }
 
+void WebContents::SetGuestSize(int width, int height) {
+  SetSizeParams params;
+  params.normal_size.reset(new gfx::Size(width, height));
+  SetSize(params);
```

Comment:
@kevinsawicki `SetGuestSize` would be a no-op for `BrowserWindow` web contents because it ends up calling `SetSize`, which checks for `guest_delegate_`. I guess we could also document `setSize` instead of adding `setGuestSize`. What do you think?